README.md

```diff
@@ -0,0 +1,37 @@
+# GSI-fix
+Static ASCII files used in the [GSI](https://github.com/NOAA-EMC/GSI) by various applications to describe the characteristics of the system.
```

Comment:
The ASCII files are actually less static than the binary GSI fix files.   Changes to binary GSI fix files are less common.  Changes to ASCII GSI fix files are more common given the uses for these fix files as stated on lines 4 to 9.   We created GSI-fix to track changes in ASCII GSI-fix files.

Suggestion:  remove `Static` and simply start with `ASCII files ...`